package.json

```diff
@@ -160,6 +161,9 @@
         "packages/**/*.d.ts"
       ]
     },
+    "chromatic": {
+      "command": "chromatic --build-script-name storybook -o dist --exit-zero-on-changes"
```

Comment:
If it has no wireit dependencies, then it's fine to have the command directly in the `"scripts"` section (similar to `extract-i18n`)